### ISSUE TITLE
Node.js 16 actions are deprecated

### ISSUE THREAD

**Issue by druzsan** [LanceMcCarthy/Action-AzureBlobUpload#403]
Recently GitHub started to warn about usage of Node.js 16 actions in workflows:
> Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: thollander/actions-comment-pull-request@v2. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.

**Comment by LanceMcCarthy** [comment]
Thanks @druzsan, I'll update ASAP.

**Comment by LanceMcCarthy** [comment]
@druzsan I've published `v3.0.0` (which can also be referenced as `v3`), please try it an let me know if you have any problems

```
- uses: LanceMcCarthy/Action-AzureBlobUpload@v3
  ...
  ...
```

**Comment by druzsan** [comment]
@LanceMcCarthy it worked just fine!
Thank you for a quick response and update.
